test/account_api_test.rb

```diff
@@ -308,6 +308,31 @@
         api_client.get_attributes_names(govuk_account_session: govuk_account_session, attributes: [attribute_name])
       end
     end
+
+    describe "#delete_user" do
+      let(:path) { "/api/user" }
+
+      it "responds with 204 OK if the user existed and has been deleted" do
+        binding.pry
```

Comment:
Think this is causing your build to fail 😬

Reply:
ah... i thought it was the pact tests waiting for t[his to get merged](https://github.com/alphagov/account-api/pull/154),
woops! thanks!